### PR TITLE
Preparations for validate futureWallet's gasToken

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-relayer/test/core/services/execution/PendingMessages.test.ts [hunk 1]
```diff
@@ -110,15 +110,15 @@ describe('INT: PendingMessages', () => {
       await pendingMessages.add(signedMessage);
       await messageRepository.markAsPending(messageHash, '0x829751e6e6b484a2128924ce59c2ff518acf07fd345831f0328d117dfac30cec', '2020');
       const status = await pendingMessages.getStatus(messageHash);
-      const required = await (await walletContractService.getRequiredSignatures(signedMessage.from)).toNumber();
+      const required = (await walletContractService.getRequiredSignatures(signedMessage.from)).toNumber();
```
Comment (JustynaBroniszewska) [inline universal-login-relayer/test/core/services/execution/PendingMessages.test.ts:113]: why duble await?

Comment (xitronix) [reply universal-login-relayer/test/core/services/execution/PendingMessages.test.ts:113]: I don't know. I reduced it

File: universal-login-commons/src/core/utils/safeMultiply.ts [hunk 2]
```diff
@@ -8,6 +8,12 @@ export const safeMultiplyAndFormatEther = (valueInWei: utils.BigNumber, multipli
   return utils.formatUnits(resultE18, SAFE_MULTIPLY_UNITS_NORMALIZE);
 };
 
+export const safeMultiply = (valueInWei: utils.BigNumber, multiplier: utils.BigNumberish) => {
```
Comment (JustynaBroniszewska) [inline universal-login-commons/src/core/utils/safeMultiply.ts:11]: `safeMultiplyAndFormatEther` isn' the same?

File: universal-login-relayer/test/core/services/execution/PendingMessages.test.ts (hunk 1 above)
Comment (JustynaBroniszewska) [reply universal-login-relayer/test/core/services/execution/PendingMessages.test.ts:113]: Aaaa sorry, I thought that you added the second await 😅

**Pull request merged**